README.md

```diff
@@ -1 +1,63 @@
-# Hello-Microverse
\ No newline at end of file
+# :green_book: Table of Contents
+- [:book: About the Project](#about-project)
+  - [:hammer_and_wrench: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [:busts_in_silhouette: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+# :book: [Hello Microverse project]
+**[Hello Microverse project]** is a simple project without any coding complex built with html and css that display a hello microverse message in your browser.
+## :hammer_and_wrench: Built With
+### Tech Stack
+**Client**
+React.js
+**Server**
+Express.js
+**Database**
+PostgreSQL
+### Key Features
+- **[When clicked it prints "Hello Microverse" in aquamarine color]**
+## :rocket: Live Demo
+- [No Live Demo Link]
+## :computer: Getting Started
+prints Hello Microverse in default browser.
+### Prerequisites
+Clone this repository to your desired folder and open with any browser.
+## :busts_in_silhouette: Authors
+:bust_in_silhouette: **Emmanuel Chijioke**
+- GitHub: [@AceMannie](https://github.com/AceMannie)
+- Twitter:[@acemanniee](https://twitter.com/acemanniee)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emmanuel-chijioke-34383b261/)
+## :telescope: Future Features
+- [ ] **[Add more forms]**
+- [ ] **[Make an atractive Layout]**
+- [ ] **[Add some Javascript]**
+## :handshake: Contributing
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
+## :star:️ Show your support
+If you like this project feel free to share with your friends.
+## :pray: Acknowledgments
+I would like to thank Microverse for providing the resources and my coding partners for the support in making this project a success.
+## :memo: License
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```

Comment:
- [ ] Kindly create a personalized License file that you can link to in your project instead of directing users to a general license template from `choosealicense.com`.